pf4j/src/main/java/org/pf4j/ClassLoadingSource.java

```diff
@@ -0,0 +1,5 @@
+package org.pf4j;
+
+public enum ClassLoadingSource {
```

Comment:
Please add the copyright header (copy/paste from other classes) and a short description (javadoc).

Reply:
done